docs/tools/vdb_table/data/txtai.json

```diff
@@ -113,19 +113,19 @@
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "txtai embeddings indexes can be loaded and run directly in a Python application"
   },
   "multi_tenancy": {
     "support": "",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "full",
```

Comment:
The intent of this attribute is to highlight VDBs that allow the index to be kept in disk while operational. I'll clarify that in [this line](https://github.com/superlinked/VectorHub/blob/80c46c947f1a7f8623303bb2320bd131c7fc6e15/docs/tools/vdb_table/vendor.schema.json#L33C89-L33C150).

I believe your comment is only around taking a backup of the in-RAM index?

Reply:
Thank you for the clarification. 

txtai does support Faiss mmap indexes. I would consider this an on disk index but if it's not in your definition, I'm fine changing it as you see fit. 

- https://neuml.github.io/txtai/embeddings/configuration/ann/#faiss
- https://github.com/facebookresearch/faiss/wiki/Indexes-that-do-not-fit-in-RAM#loading-a-standard-index-on-disk

Reply:
Thanks for providing that context. Please mark this as "partial", with the explanation in comment, and add the first link as source_url.

Everything else looks good.

Reply:
Seems fair, done.